Src/IronPython/Runtime/MemoryView.cs

```diff
@@ -265,12 +265,22 @@ public sealed class MemoryView : ICodeFormattable, IWeakReferenceable {
         }
 
         private byte[] unpackBytes(string format, object o) {
-            if (TypecodeOps.TryGetBytes(format, o, out byte[] bytes)) {
-                return bytes;
-            } else if (o is Bytes b) {
-                return b._bytes; // CData returns a bytes object for its type
-            } else {
-                throw PythonOps.NotImplementedError("No conversion for type {0} to byte array", PythonOps.GetPythonTypeName(o));
+            try {
+                if (TypecodeOps.TryGetBytes(format, o, out byte[] bytes)) {
+                    return bytes;
+                } else if (o is Bytes b) {
+                    return b._bytes; // CData returns a bytes object for its type
+                } else {
+                    throw PythonOps.NotImplementedError("No conversion for type {0} to byte array", PythonOps.GetPythonTypeName(o));
+                }
+            } catch (OverflowException) {
+                bool valueError = Converter.TryConvertToBigInteger(o, out _);
+
+                if (valueError) {
+                    throw PythonOps.ValueError("memoryview: invalid type for format '{0}'", format);
```

Comment:
Should this read "invalid value"  instead of "invalid type"?

Reply:
Changed.